modules/private-clusters-about-aws.adoc

```diff
@@ -24,5 +24,5 @@ The installation program does use the `baseDomain` that you specify to create a
 
 The ability to add public functionality to a private cluster is limited.
 
-* You cannot make the Kubernetes API endpoints public after installation without taking additional actions, including creating public subnets in the VPC for each availability zone in use, creating a public load balancer, and configuring the control plane security groups to allow traffic from Internet on 6443 (Kubernetes API port).
+* You cannot make the Kubernetes API endpoints public after installation without taking additional actions, including creating public subnets in the VPC for each availability zone in use, creating a public load balancer, and configuring the control plane security groups to allow traffic from internet on 6443 (Kubernetes API port).
```

Comment:
"from _the_ internet"? If so, might as well fix this while here. :) Looks like it's duplicated a few times as well.

Reply:
Oh, good point! Thank you. :)